internal/store/fixtures.go

```diff
@@ -0,0 +1,124 @@
+package store
+
+import (
+	"time"
+
+	"github.com/google/uuid"
+	"github.com/osbuild/osbuild-composer/internal/blueprint"
+	"github.com/osbuild/osbuild-composer/internal/common"
+	"github.com/osbuild/osbuild-composer/internal/compose"
+	"github.com/osbuild/osbuild-composer/internal/target"
+)
+
+func FixtureBase() *Store {
```

Comment:
It's a bit odd to have the fixtures be public types on the store. Is the idea to make them private at some point?

Reply:
Not sure about that. I think keeping them public is in keeping with the way the fixtures are meant to be used, but I'm happy to discuss that in a follow-up, unless you think it is a blocker?